fineract-provider/config/checkstyle/checkstyle.xml

```diff
@@ -46,6 +46,11 @@
 
 
     <module name="TreeWalker">
+    <module name="ParameterName">
+            <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9]*$|[a-z]"/>
```

Comment:
@thesmallstar I was surprised that the RegExp change was required.. * is zero or many (+ is "at least one"), are you 100% sure that this is required? I'm just going to merge this anyway as-is, but perhaps you'd like to retry after if this can't be changed back?